### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.intellij.diagnostic.PluginException: 303 ms to call on EDT ServerExplorerToolWindowFactory$RefreshAllAction#update@ToolwindowTitle (com.microsoft.intellij.ui.ServerExplorerToolWindowFactory$RefreshAllAction). Revise AnAction.getActionUpdateThread property [Plugin: com.microsoft.tooling.msservices.intellij.azure]

### ISSUE THREAD

**Issue by jishanhk** [microsoft/azure-tools-for-java#7421]
IntelliJ build version: 2022.3.2 IU-223.8617.56
OS: Windows 10
JDK: JetBrains s.r.o. 17.0.5
Plugin version: 3.74.1-2022.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,7,7,1537x824,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.intellij.diagnostic.PluginException: 303 ms to call on EDT ServerExplorerToolWindowFactory$RefreshAllAction#update@ToolwindowTitle (com.microsoft.intellij.ui.ServerExplorerToolWindowFactory$RefreshAllAction). Revise AnAction.getActionUpdateThread property [Plugin: com.microsoft.tooling.msservices.intellij.azure]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
		at com.microsoft.azure.toolkit.lib.auth.Account.getSelectedSubscriptions(Account.java:222)
		at com.microsoft.azure.toolkit.lib.auth.Account.isLoggedInCompletely(Account.java:239)
		at com

**Comment by wangmingliang-ms** [comment]
this issue should have been fixed in the latest v3.77.0. Feel free to reopen it if you encounter this problem again.

**Comment by wangmingliang-ms** [comment]
this issue should have been fixed in the latest v3.77.0. Feel free to reopen it if you encounter this problem again.
